woocommerce/woocommerce
# Merge React Admin

### Changes proposed in this Pull Request:

This PR moves the entire `woocommerce-admin` folder to `woocommerce/client/admin`.

We are seeing markdown linting errors because all of the `md` content in the package is being passed to the linter as "changed" markdown.

We are seeing PHP linting errors because the strict types logic does not apply the exclusions in `phpcs.xml` and `git mv`'ed files are deleted in the old location and created in the new location.

Closes #38378 .



### How to test the changes in this Pull Request:



Using the [WooCommerce Testing Instructions Guide](https://github.com/woocommerce/woocommerce/wiki/Writing-high-quality-testing-instructions), include your detailed testing instructions:

1. In the root of the repo run `pnpm clean && pnpm i`
2. In the `plugins/woocommerce` folder run `pnpm watch:build`
3. Edit some of the `label` properties in `plugins/woocommerce/client/admin/client/analytics/report/revenue/config.js`
4. Wait for the `watch:build` to log `watch:build:project:copy-assets: [nodemon] clean exit - waiting for changes before restart` with no further activity
5. Go to Analytics > Revenue
6. The labels in the header section should reflect your edits (hard refresh may be necessary)
7. Edit `plugins/woocommerce/client/admin/client/core-profiler/style.scss`
8. Change the `background-color` for `.woocommerce-profile-wizard__body` (~line 10)
9. Wait for the `watch:build`
10. Go to WooCommerce > Orders > Help > Setup Wizard
11. Click `Setup Wizard`
12. The store profiler page should have the background color you provided (hard refresh may be necessary)
13. Check that the new product editor and the analytics reports work correctly.



### Changelog entry







<details>

<summary>Changelog Entry Details</summary>

#### Significance




#### Type




#### Message 

</details>

<details>

<summary>Changelog Entry Comment</summary>

#### Comment 

</details>
